recipes/ckanparams.rb

```diff
@@ -0,0 +1,38 @@
+#
+# Author:: Carl Antuar (<carl.antuar@smartservice.qld.gov.au>)
+# Cookbook Name:: datashades
+# Recipe:: ckanparams
+#
+# Defines some default parameters for CKAN deployments.
+#
+# Copyright 2024, Queensland Government
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Obtain some stack attributes for the recipes to use
+#
+
+include_recipe "datashades::stackparams"
+
+# Retrieve attributes from SSM Parameter Store
+node.default['datashades']['log_bucket'] = `aws ssm get-parameter --region "#{node['datashades']['region']}" --name "/config/CKAN/#{node['datashades']['version']}/common/s3LogsBucket" --query "Parameter.Value" --output text`.strip
```

Comment:
Should be SOLR be pushing its log here also?

Reply:
You're right. I'll have to update the policy so Solr can access 'common' Parameter Store values.